tasks/scripts/integration

```diff
@@ -25,7 +25,7 @@ image_ref () {
 }
 
 docker load -i dev-image/image.tar
-export TEST_CONCOURSE_DEV_IMAGE="$(image_ref dev-image concourse/dev)"
+export TEST_CONCOURSE_DEV_IMAGE="$(image_ref dev-image "$(cat dev-image/repository)")"
```

Comment:
Makes sense, though this assumes the `repository` file exists - it won't if the input was generated by the `oci-build-task` (i.e. in a PR build).

I think we can push this logic into `image_ref`:

```suggestion
export TEST_CONCOURSE_DEV_IMAGE="$(image_ref dev-image)"
```

and then change L21 in `image_ref` to this:

```bash
    echo "$(cat ${dir}/repository):$(cat ${dir}/tag)"
```

L23 in `image_ref` seems a little sketchy, but that can be out of scope of this PR. It looks like it just sets the value to a naked digest, which doesn't work with `docker-compose` on my machine. Which begs the question - how is the PR build passing? 🤔 Maybe we can change it so that it gives the digest a random local name and then returns `${local_name}:$(cat ${dir}/digest)`. (Again, just thinking out loud, no need to address this here.)

Reply:
I've pushed the repository lookup into `image_ref`, this also allowed us to remove the second argument for all calls to `image_ref`.